lib/OrdinaryDiffEqPDIRK/src/algorithms.jl

```diff
@@ -1,10 +1,26 @@
-################################################################################
-
-#################################################
-"""
-PDIRK44: Parallel Diagonally Implicit Runge-Kutta Method
-A 2 processor 4th order diagonally non-adaptive implicit method.
-"""
+@doc differentiation_rk_docstring(
+    "A 2 processor 4th order diagonally non-adaptive implicit method.",
+    "PDIRK44",
+    "Parallel Diagonally Implicit Runge-Kutta Method.";
+    references = """"@article{iserles1990theory,
+    title={On the theory of parallel Runge—Kutta methods},
+    author={Iserles, Arieh and Norrsett, SP},
+    journal={IMA Journal of numerical Analysis},
+    volume={10},
+    number={4},
+    pages={463--488},
+    year={1990},
+    publisher={Oxford University Press}}""",
+    extra_keyword_description = """
+    - `nlsolve`: TBD,
+    - `extrapolant`: TBD,
+    - `threading`: TBD,
+    """,
+    extra_keyword_default = """
+    nlsolve = NLNewton(),
+    extrapolant = :constant,
```

Comment:
@ChrisRackauckas similar comment to the controller, it is hard to know what other values might be allowed, since `extrapolant` does not seem to take local action in `OrdinaryDiffEqPDIRK`.

Reply:
Some methods have a specialized controller that is highly recommended to be used. Other controllers will still work, but not as well.